src/webapp/azext_webapp/custom.py

```diff
@@ -308,3 +310,75 @@ def _should_create_new_asp(cmd, rg_name, asp_name, location):
                 item.location == location):
             return False
     return True
+
+
+def enable_one_deploy(cmd, resource_group_name, name, src, deploy_type=None, is_async=None, target_path=None, timeout=None, slot=None):
+    import ntpath
+    logger.warning("Preparing for deployment")
+    user_name, password = _get_site_credential(cmd.cli_ctx, resource_group_name, name, slot)
+
+    try:
+        scm_url = _get_scm_url(cmd, resource_group_name, name, slot)
+    except ValueError:
+        raise CLIError('Failed to fetch scm url for function app')
```

Comment:
why does this say function app here? can you update as failed to fetch sam url for app {} -> show app name as well.